.github/workflows/test.yml

```
@@ -0,0 +1,33 @@
+name: Run unit and ui tests
+
+on:
+  push:
```

```suggestion
  push:
    branches-ignore:
      - 'develop'
      - 'main'
```

For this branches a push triggers a release. The execution of the test should be added to the publish workflow.
(also see the android test and publish workflow)